README.md

```diff
@@ -1,3 +1,119 @@
+# Penumbra Web Monorepo
```

Comment:
todo: remove the updates to Readme.md. Was that you local-only documentation? We'd definitely need to revise this readme, but I believe the main focus would be on the exported packages, not on the project structure. Feel free to update the UI or Minifront readme though.